miqa/core/rest/other_endpoints.py

```diff
@@ -6,6 +8,11 @@
 
 from miqa.core.models.scan_decision import ArtifactState, default_identified_artifacts
 
+MIQA_VERSION = subprocess.run(
+    ['git', 'describe', '--tags'],
```

Comment:
@annehaley this would always generate the last tag available on the master branch. Wouldn't it be better if we have the SHA that was used to build the image?

Reply:
Added in [e2ec803](https://github.com/OpenImaging/miqa/pull/486/commits/e2ec803b034eac53c42e80977447951e23261029)